cdap-ui/app/cdap/services/helpers.js

```diff
@@ -284,7 +284,11 @@ const composeEnhancers = (storeTitle) =>
       name: storeTitle
     }) : compose;
 
-const reverseWithoutMutating = (array) => {
+const reverseArrayWithoutMutating = (array) => {
+  if (isNil(array)) {
+    return null;
```

Comment:
Might want to return `[]`. The other return statement is return an array. It would be easier to assume if this guy also returns the same I guess?